src/style.css

```
@@ -1,3 +1,70 @@
 body {
+  --color: #00acdc;
+  
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100% - 100px;
```

Fix it and add `box-sizing: border-box` to the `*` selector
```suggestion
  width: 100%;
```